Cargo.toml

```diff
@@ -3,6 +3,10 @@ members = ["quinn", "quinn-proto", "quinn-udp", "bench", "perf", "fuzz"]
 default-members = ["quinn", "quinn-proto", "quinn-udp", "bench", "perf"]
 resolver = "2"
 
+[workspace.dependencies]
```

Comment:
Would you mind doing the two steps in two commits? Makes it a lot easier to review.

Reply:
Done. Mind taking another look @djc?